src/components/atoms/ActionButton.js

```diff
@@ -5,16 +5,35 @@ import PropTypes from "prop-types";
  * Button component which is used to trigger actions on the page
  */
 export function ActionButton(props) {
+  /**
+   * This function checks if the button is a filter button
+   */
+  const isFilterButton = (type, count) => {
+    if (type === "filter")
+      return (
+        <div className="ml-4 rounded-full h-6 w-6 flex item-center justify-center bg-gray-300 text-gray-700">
```

Comment:
Perhaps separate this out into another component so its cleaner

Reply:
The counter is now a separate component